### PR TITLE
Preetika _assignment-1

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Smit-create** [comment]
Why have you used two files, i.e `Assignment-1.ipynb` and `users.ipynb`?

**Comment by ghost** [comment]
That was done unknowingly... I used the file and then saved it with another name not intentional.

File: Assignment/users.ipynb [hunk 1]
```diff
@@ -0,0 +1,3398 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#your code here task 2"
+   ]
```
Comment (sin-ha) [inline Assignment/users.ipynb:1280]: I guess this is empty.

**Review by sin-ha — APPROVED** [review]
The pr looks good else for what @Smit-create has said

File: Assignment/users.ipynb (hunk 1 above)
Comment (ghost) [reply Assignment/users.ipynb:1280]: I was not  able to solve it. I tried but i thought i would refer to the answers once they were given(smit bhaiya said answers will be provided or hints maybe) so i did not try any longer

**Pull request closed without merging**